### ISSUE TITLE
Headings are too large

### ISSUE THREAD

**Issue by Abhayakara** [rfc-format/draft-iab-rfc-css-bis#1]
I think it's good to use larger fonts for headings, but the relative size needs to be reduced substantially.   And it's not clear to me that there's a good reason for different level headings to be different sizes.   The current header sizes really waste a lot of vertical space, so that on the first page you see hardly any actual document text.

**Comment by martinthomson** [comment]
I disagree, but then I did the styles for [this](https://quicwg.github.io/base-drafts/draft-ietf-quic-tls.html).  It might depend on your usage model.

> on the first page you see hardly any actual document text

... so no real change then :)

**Comment by martinthomson** [comment]
Actually, I think that I agree with Ted here.  On smaller screens, the headings really are too large.  Try enabling responsive design mode and picking a very small screen.  On an iPhone 5s, the title just about fills the entire page.

On a very large screen, I find that large headings are useful, so maybe there's a compromise here somewhere.

**Comment by Abhayakara** [comment]
I was reading the text on a 32"  4k monitor, and the heads still looked too
big.   The heads are just there for navigation: the content under them is
what's important.   So they only need to stand out enough to allow the
reader to navigate.

**Comment by hildjj** [comment]
It's also noticeable when I zoom in to increase the font size to something I can read more easily, so I think I agree that the size of the main text and the headings should be closer to one another.

**Comment by hlflanagan** [comment]
I'm suggesting for the next version of the draft CSS to leave the heading sizes as they are to make sure there is a very clear distinction, given we're going to remove the text indentation outside of lists. We will have a chance to discuss when we have the example ready to look at.
